src/database/DataTypes/Period.js

```diff
@@ -62,6 +62,10 @@ export class Period extends Realm.Object {
     const endDate = moment(this.endDate).format('DD/MM/YY');
     return `${startDate} - ${endDate}`;
   }
+
+  get numberOfDays() {
+    return moment(this.endDate).diff(moment(this.startDate), 'days');
```

Comment:
Just want to check with you, is the behaviour of this:

1. 1/1/2020 to 1/1/2020 = 1 day (as opposed to 0 days)
2. 1/1/2020 to 31/1/2020 = 31 days (as opposed to 30)

Reply:
> 1/1/2020 to 1/1/2020 = 1 day (as opposed to 0 days)

0 days

> 1/1/2020 to 31/1/2020 = 31 days (as opposed to 30)

31 days

Reply:
Ignoring the fact I just approved this... It needs to return 1 day for a period that starts/ends on the same date.